p2panda-sync/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "p2panda-sync"
+version = "0.1.0"
+edition = "2021"
+
+[features]
+default = ["protocols"]
+protocols = ["dep:p2panda-core", "dep:p2panda-store", "dep:ciborium", "core"]
```

Comment:
`protocols` is maybe a too generic name for a feature flag? How about `log-height`?

Reply:
Nice, I added a new module `protocols` and placed `log_height.rs` in there. We can discuss if this is where we want to introduce protocol implementations, or if we'd rather they were in their own module.

Reply:
I find the word protocols somewhat to big, why not a dedicated log-height module, set-recon module etc.?